streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -39,13 +39,21 @@ public Long apply() {
             return 0L;
         }
     };
+
     final Aggregator<K, V, Long> countAggregator = new Aggregator<K, V, Long>() {
         @Override
         public Long apply(K aggKey, V value, Long aggregate) {
             return aggregate + 1;
         }
     };
 
+    final Initializer<V> reduceInitializer = new Initializer<V>() {
```

Comment:
This is moved from TimeWindowedStreamImpl, just to be consistent with the other const functions.